<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSBasicTransitions</class>
 <widget class="QWidget" name="OBSBasicTransitions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>127</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="transitionsFrame">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="transitions">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="accessibleName">
         <string>Transition</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="transitionDurationLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Basic.TransitionDuration</string>
          </property>
          <property name="buddy">
           <cstring>transitionDuration</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="transitionDuration">
          <property name="accessibleName">
           <string>Basic.TransitionDuration</string>
          </property>
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>20000</number>
          </property>
          <property name="singleStep">
           <number>50</number>
          </property>
          <property name="value">
           <number>300</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="transitionAdd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Basic.AddTransition</string>
          </property>
          <property name="accessibleName">
           <string>Basic.AddTransition</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="obs.qrc">
            <normaloff>:/res/images/plus.svg</normaloff>:/res/images/plus.svg</iconset>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="class" stdset="0">
           <string notr="true">btn-tool icon-plus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="transitionRemove">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Basic.RemoveTransition</string>
          </property>
          <property name="accessibleName">
           <string>Basic.RemoveTransition</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="obs.qrc">
            <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="class" stdset="0">
           <string notr="true">btn-tool icon-trash</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="transitionProps">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Basic.TransitionProperties</string>
          </property>
          <property name="accessibleName">
           <string>Basic.TransitionProperties</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="obs.qrc">
            <normaloff>:/settings/images/settings/general.svg</normaloff>:/settings/images/settings/general.svg</iconset>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="class" stdset="0">
           <string notr="true">btn-tool icon-dots-vert</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections/>
</ui>
